hack/make/test-integration-flaky

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env bash
+set -e -o pipefail
+
+source hack/validate/.validate
+new_tests=$(
+    validate_diff --diff-filter=ACMR --unified=0 -- 'integration/*_test.go' |
+    grep -E '^(\+func )(.*)(\*testing)' || true
+)
+
+if [[ -z "$new_tests" ]]; then
```

Comment:
oh; could you change it to single brackets? see  https://github.com/moby/moby/pull/38527 (https://github.com/moby/moby/pull/38424#issuecomment-452882054)